src/main/java/ch/naviqore/gtfs/schedule/model/Stop.java

```diff
@@ -17,6 +18,7 @@ public final class Stop implements Initializable, Location<GeoCoordinate> {
 
     private final String id;
     private final String name;
+    @Nullable private final String parentId;
```

Comment:
And should we track it only this way around?

Stop --> points to Parent?

Or should we also track the childs on the parent? I am not sure if we need this in the router? But I assume it would be nice for the service to list only the parents or unique stops.

Reply:
This is done in NAV-87. The service has a new HashMap where each ParentStopId is linked to a list of all related stops (including itself).